### ISSUE TITLE
Could not work

### ISSUE THREAD

**Issue by Synoopt** [kyab/MySpleeter#7]
I paste the whole report here, dont know why its like this.
```
separate_to_file(/Users/zyd/Downloads/C/51.mp4,/Users/zyd/Downloads/C) failed : Error while reading resource variable conv2d_27/kernel from Container: localhost. This could mean that the variable was uninitialized. Not found: Container localhost does not exist. (Could not find resource: localhost/conv2d_27/kernel)
[[node conv2d_27/Conv2D/ReadVariableOp (defined at /tensorflow_core/python/framework/ops.py:1748) ]]

Original stack trace for 'conv2d_27/Conv2D/ReadVariableOp':
File "/spleeter/separator.py", line 228, in separate_to_file
sources = self.separate(waveform, audio_descriptor)
File "/spleeter/separator.py", line 195, in separate
return self._separate_librosa(waveform, audio_descriptor)
File "/spleeter/separator.py", line 173, in _separate_librosa
outputs = self._get_builder().outputs
File "/spleeter/model/__init__.py", line 301, in outputs
self._build_outputs()
File "/spleeter/model/__init__.py", line 476, in _build_outputs
self._outputs = self.masked_stfts
File "/spleeter/model/__init__.py", line 325, in masked_stfts
self._build_masked_stfts()
File "/spleeter/model/__init__.py", line 440, in _build_masked_stfts
for instrument, mask in self.masks.items():
File "/spleeter/model/__init__.py", line 319, in masks
self._build_masks()
File "/spleeter/model/__init__.py", line 409, in _build_masks
output_dict = self.model_outputs
File "/spleeter/model/__init__.py", line 295, in model_outputs
self._build_model_outputs()
File "/spleeter/model/__init__.py", line 206, in _build_model_outputs
self._params['model']['params'])
File "/spleeter/model/functions/unet.py", line 194, in softmax_unet
output_mask_logit=True))
File "/spleeter/model/functions/unet.py", line 170, in apply_unet
kernel_initializer=kernel_initializer)((batch12))
File "/tensorflow_core/python/keras/engine/base_layer.py", line 854, in __call__
outputs = call_fn(cast_inputs, *args, **kwargs)
File "/tensorflow_core/python/keras/layers/convolutional.py", line 197, in call
outputs = self._convolution_op(inputs, self.kernel)
File "/tensorflow_core/python/ops/nn_ops.py", line 1134, in __call__
return self.conv_op(inp, filter)
File "/tensorflow_core/python/ops/nn_ops.py", line 639, in __call__
return self.call(inp, filter)
File "/tensorflow_core/python/ops/nn_ops.py", line 624, in _with_space_to_batch_call
result = self.op(input_converted, filter)
File "/tensorflow_core/python/ops/nn_ops.py", line 238, in __call__
name=self.name)
File "/tensorflow_core/python/ops/nn_ops.py", line 2010, in conv2d
name=name)
File "/tensorflow_core/python/ops/gen_nn_ops.py", line 1071, in conv2d
data_format=data_format, dilations=dilations, name=name)
File "/tensorflow_core/python/framework/op_def_library.py", line 528, in _apply_op_helper
preferred_dtype=default_dtype)
File "/tensorflow_core/python/framework/ops.py", line 1297, in internal_convert_to_tensor
ret = conversion_func(value, dtype=dtype, name=name, as_ref=as_ref)
File "/tensorflow_core/python/ops/resource_variable_ops.py", line 1789, in _dense_var_to_tensor
return var._dense_var_to_tensor(dtype=dtype, name=name, as_ref=as_ref)  # pylint: disable=protected-access
File "/tensorflow_core/python/ops/resource_variable_ops.py", line 1214, in _dense_var_to_tensor
return self.value()
File "/tensorflow_core/python/ops/resource_variable_ops.py", line 524, in value
return self._read_variable_op()
File "/tensorflow_core/python/ops/resource_variable_ops.py", line 608, in _read_variable_op
self._dtype)
File "/tensorflow_core/python/ops/gen_resource_variable_ops.py", line 587, in read_variable_op
"ReadVariableOp", resource=resource, dtype=dtype, name=name)
File "/tensorflow_core/python/framework/op_def_library.py", line 794, in _apply_op_helper
op_def=op_def)
File "/tensorflow_core/python/util/deprecation.py", line 507, in new_func
return func(*args, **kwargs)
File "/tensorflow_core/python/framework/ops.py", line 3357, in create_op
attrs, op_def, compute_device)
File "/tensorflow_core/python/framework/ops.py", line 3426, in _create_op_internal
op_def=op_def)
File "/tensorflow_core/python/framework/ops.py", line 1748, in __init__
self._traceback = tf_stack.extract_stack()
